electionleaflets/apps/constituencies/admin.py

```diff
@@ -1,19 +0,0 @@
-from django.contrib import admin
-from constituencies.models import Constituency
-
-
-class ConstituencyOptions(admin.ModelAdmin):
-    list_display = [
-        "name",
-        "slug",
-    ]
-    search_fields = ["name"]
-    ordering = ["name"]
-
-
-class ConstituencyTypeOptions(admin.ModelAdmin):
-    list_display = ["name", "country"]
-    search_fields = ["name"]
-
-
-admin.site.register(Constituency, ConstituencyOptions)
```

Comment:
Given we are keeping the `Constituency` model, would it be a good idea to keep it registered in the admin?